Cargo.toml

```diff
@@ -15,7 +15,7 @@ async-recursion = "0.3.1"
 bincode = "1.2.1"
 bls = { package = "threshold_crypto", version = "~0.4.0" }
 bls_dkg = "~0.2.0"
-bls_signature_aggregator = { git = "https://github.com/madadam/bls_signature_aggregator.git", branch = "repetition" }
+bls_signature_aggregator = { git = "https://github.com/maidsafe/bls_signature_aggregator.git" }
```

Comment:
why not use the version number?

Reply:
Wasn't sure it's published. Will check and change.